rust/rsc/src/rsc/add_job.rs

```diff
@@ -47,61 +47,61 @@ pub async fn add_job(
                 let job = insert_job.save(txn).await?;
                 let job_id = job.id.unwrap();
 
-                let mut visible_files = Vec::new();
-                for vis_file in vis {
-                    visible_files.push(visible_file::ActiveModel {
+                let visible_files: Vec<visible_file::ActiveModel> = vis
```

Comment:
BTW, you can use `Vec<_>` here. In fact I think you might be able to get away with no type annotation because `insert_many` expects a Vec and your pre-collector iterator has the right item type

Reply:
`Vec<_>` worked but completely removing didn't

Reply:
Interesting, ok. I'm still not 100% sure how Rust type inference works. Maybe insert_many has a more general signature than last time I looked at it.

Reply:
Ah yeah, they updated it to use an iterator! You can actually avoid the collect and pass the iterator directly in and you'll be even *more* efficient. I remember the signature of that being a Vec in the previous version.